website/docs/docs/get-started/privatelink/about-private-link.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "About PrivateLink"
+id: about-private-link
+description: "Configuring PrivateLink for AWS"
+sidebar_label: "About PrivateLink"
+---
+
+## What is PrivateLink?
+
+PrivateLink enables you to connect your data warehouse (DWH) hosted on AWS, in any region, to a dbt Cloud multi-tenant environment over the AWS backbone using the PrivateLink technology. PrivateLink allows dbt Cloud customers to meet security and compliance controls as it restricts information shared between dbt Cloud and the DWH from reaching the public internet. You can read more about AWS PrivateLink technology [here](https://aws.amazon.com/privatelink/).
+
+The initial release of support for AWS PrivateLink features integration with the following data warehouse providers:
+
+- Redshift
+- Snowflake
+- Databricks
+
+## Configuring PrivateLink
```

Comment:
```suggestion
## Configure PrivateLink
```

Reply:
sugg: titles should not be a gerund. shorter for a cleaner rightbar (page toc). there are some exceptions like "Troubleshooting"

fix all in your 3 new pages